docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -322,7 +322,7 @@ Built-in features provide a limited range of [polymorphic serialization](system-
 
 Suppose, for example, you have a `Person` abstract base class, with `Employee` and `Customer` derived classes. Polymorphic deserialization means that at design time you can specify `Person` as the deserialization target, and `Customer` and `Employee` objects in the JSON are correctly deserialized at run time. During deserialization, you have to find clues that identify the required type in the JSON. The kinds of clues available vary with each scenario. For example, a discriminator property might be available or you might have to rely on the presence or absence of a particular property. The current release of `System.Text.Json` doesn't provide attributes to specify how to handle polymorphic deserialization scenarios, so custom converters are required.
 
-The following code shows a base class, two derived classes, and a custom converter for them. The converter uses a discriminator property to do polymorphic deserialization. The type discriminator isn't in the class definitions but is created during serialization and is read during deserialization.
+The following code shows a base class, two derived classes, and a custom converter for them. The converter uses a discriminator property to do polymorphic deserialization. The type discriminator isn't in the class definitions but is created during serialization and is read during deserialization. The example code requires JSON object name/value pairs to stay in order, which is not a standard requirement of JSON.
```

Comment:
```suggestion
The following code shows a base class, two derived classes, and a custom converter for them. The converter uses a discriminator property to do polymorphic deserialization. The type discriminator isn't in the class definitions but is created during serialization and is read during deserialization.

> [!IMPORTANT]
> The example code requires JSON object name/value pairs to stay in order, which is not a standard requirement of JSON.
```

I'd promote this to an alert as it is very important to consumers. This will likely be a gotcha that readers will struggle with, so we should call more attention to it.